protocol/find_coordinator_request.go

```diff
@@ -5,8 +5,8 @@ package protocol
 type CoordinatorType int8
 
 const (
-	CoordinatorGroup       CoordinatorType = 0
-	CoordinatorTransaction CoordinatorType = 0
+	CoordinatorGroup CoordinatorType = iota
+	CoordinatorTransaction
 )
```

Comment:
@pascaldekloe I was mentioning this change, from my just poorly acquired Go skills this would mean `CoordinatorGroup = 0` and `CoordinatorTransaction = 1`, but I couldn't find any reference to it on the protocol page. It does make a lot of sense as before both Transaction and Group coordinators had the same `key_type` which is weird.

Reply:
The value is not documented indeed. Let's use the Java source code then.
 https://github.com/apache/kafka/blob/407bcdf78e06f83f2b358d2cbd96aed348a5c28f/clients/src/main/java/org/apache/kafka/common/requests/FindCoordinatorRequest.java#L95